### PR TITLE
fix: isReadOnly in Autocomplete MDX

### PR DESCRIPTION
**Pull request by alphaxek**
Closes https://github.com/nextui-org/nextui/issues/2420

## 📝 Description

- revised the isReadOnly logic in Autocomplete
- created isReadOnly example in storybook

## ⛳️ Current behavior (updates)
when isReadOnly is set, users

- cannot edit the input
- can open the listbox
- can select the menu option

## 🚀 New behavior
when isReadOnly is set, users

- cannot edit the input
- can open the listbox
- cannot select the menu option

## 💣 Is this a breaking change (Yes/No):
No

## 📝 Additional Information
When someone prefer to not open the listbox, then they should use isDisabled instead.

### CODE REVIEW

**Review by wingkwong — CHANGES_REQUESTED** [review]
I've made the [PR](https://github.com/nextui-org/nextui/pull/2429) 3 days ago. You don't need to copy my changes and add your own. You can simply supplement the doc part in this PR. In this case, a changeset is not required.

File: apps/docs/content/docs/components/autocomplete.mdx [hunk 1]
```diff
@@ -68,6 +68,13 @@ the end of the label and the autocomplete will be required.
 
 <CodeDemo title="Required" highlightedLines="8" files={autocompleteContent.required} />
 
+### Read Only
+
+If you pass the `isReadOnly` property to the autocomplete, the listbox will open to display
```
Comment (jrgarciadev) [inline apps/docs/content/docs/components/autocomplete.mdx:73]:
```suggestion
If you pass the `isReadOnly` property to the Autocomplete, the Listbox will open to display 
```

**Review by jrgarciadev — CHANGES_REQUESTED** [review]
Hey @alphaxek please update this PR with the latest `main` changes

**Comment by alphaxek** [comment]
> I've made the [PR](https://github.com/nextui-org/nextui/pull/2429) 3 days ago. You don't need to copy my changes and add your own. You can simply supplement the doc part in this PR. In this case, a changeset is not required.

Hi @wingkwong , I apologize I actually was trying to add my changes to your PR, but since it was my first time doing dual PR, I kind of messed it up, I didn't think it was going to create a whole new PR, I didn't intent to, if you want I can take it back, let me know, thanks

**Review by jrgarciadev — APPROVED** [review]
Thanks! @alphaxek

**Pull request merged**